java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -849,6 +850,118 @@ public static CompletableFuture<Long> forget(
         return executeCommand(client, new GlideString[] {gs(JSON_FORGET), key, path});
     }
 
+    /**
+     * Toggles a Boolean value stored at the root within the JSON document stored at <code>key</code>.
+     *
+     * @param client The client to execute the command.
+     * @param key The key of the JSON document.
+     * @return Returns an <code>Object[]</code> with a list of integers for the root, with the toggled
+     *     boolean value, or <code>null</code> for JSON values matching the root that are not boolean.
+     *     If <code>key</code> doesn't exist, returns <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * Json.set(client, "doc", ".", true).get();
+     * var res = Json.toggle(client, "doc").get();
+     * assert res.equals(false);
+     * res = Json.toggle(client, "doc").get();
+     * assert res.equals(true);
+     * }</pre>
+     */
+    public static CompletableFuture<Object> toggle(@NonNull BaseClient client, @NonNull String key) {
+        return executeCommand(client, new String[] {JSON_TOGGLE, key});
+    }
+
+    /**
+     * Toggles a Boolean value stored at the root within the JSON document stored at <code>key</code>.
+     *
+     * @param client The client to execute the command.
+     * @param key The key of the JSON document.
+     * @return Returns an <code>Object[]</code> with a list of integers for the root, with the toggled
+     *     boolean value, or <code>null</code> for JSON values matching the root that are not boolean.
+     *     If <code>key</code> doesn't exist, returns <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * Json.set(client, "doc", ".", true).get();
+     * var res = Json.toggle(client, gs("doc")).get();
+     * assert res.equals(false);
+     * res = Json.toggle(client, gs("doc")).get();
+     * assert res.equals(true);
```

Comment:
same here

Reply:
I have updated the documentation to remove the "array" and change the return type from `Object` to `Boolean`. For the examples, I will leave it as it for easier to read.